src/TSTransformer/macros/propertyCallMacros.ts

```diff
@@ -435,6 +435,18 @@ const READONLY_ARRAY_METHODS: MacroList<PropertyCallMacro> = {
 	findIndex: runtimeLib("array_findIndex"),
 	indexOf: runtimeLib("array_indexOf"),
 	find: runtimeLib("array_find"),
+	sort: (state, node, expression) => {
+		const list = lua.list.make(expression);
+
+		if (node.arguments?.length > 0) {
```

Comment:
```suggestion
		if (node.arguments.length > 0) {
```
I don't think the `?` is necessary here. `node.arguments` should always be defined?

Reply:
Sort can be called without arguments correct?

Reply:
Arguments will be set to an empty array if none.